### ISSUE TITLE
Broken links

### ISSUE THREAD

**Issue by ThatOneCalculator** [Waffelo/waffelo.net#1]
/projects gives a blank page and /art and /other give 404 messages

**Comment by Waffelo** [comment]
Yeah I know about that, I didn't even bother finishing it since I will try to make a new website soon

**Comment by ThatOneCalculator** [comment]
gay gay homosexual gay
